### PR TITLE
fix: many parse errors

### PR DESCRIPTION
**Pull request by brennanjl**
Resolves all errors in https://github.com/kwilteam/kwil-db/issues/896

### CODE REVIEW

**Comment by brennanjl** [comment]
Also fixes https://github.com/kwilteam/kuneiform-ls/issues/10. @charithabandi once this is merged we should update the language server

**Comment by brennanjl** [comment]
Also resolves https://github.com/kwilteam/kwil-db/issues/907

**Comment by jchappelow** [comment]
@charithabandi Testing language server required updating ls deps and rebuilding the extension?  Is it a quick test?

File: internal/engine/generate/utils.go [hunk 1]
```diff
@@ -18,33 +18,37 @@ func attributeToSQLString(colName string, attr *types.Attribute) (string, error)
 	case types.UNIQUE:
 		return "UNIQUE", nil
 	case types.MIN:
-		intVal, err := conv.Int(attr.Value)
-		if err != nil {
-			return "", err
-		}
-
-		return "CHECK (" + colName + " >= " + fmt.Sprint(intVal) + ")", nil
```
Comment (jchappelow) [inline internal/engine/generate/utils.go:26]: That's kinda funny.  I missed this pointless convert/back-convert before.    We have the same `fmt.Sprint(intVal)`  in `case types.MIN_LENGTH` too.  Might as well simplify the others?

Comment (brennanjl) [reply internal/engine/generate/utils.go:26]: Yep will do.

> I missed this pointless convert/back-convert before

This code is so old, I'm pretty sure it was written before Gavin was even on the team. I'm fairly confident this is the first time it has been reviewed ever.

**Review by Yaiba — APPROVED** [review]
LGTM.

**Pull request merged**